src/utils/openai.ts

```diff
@@ -1,7 +1,8 @@
 import https from 'https';
 import type { ClientRequest, IncomingMessage } from 'http';
 import type { CreateChatCompletionRequest, CreateChatCompletionResponse } from 'openai';
-import { type TiktokenModel } from '@dqbd/tiktoken';
+// eslint-disable-next-line camelcase
+import { TiktokenModel, encoding_for_model } from '@dqbd/tiktoken';
```

Comment:
Can you break up each named import into its own line so the eslint disable directive can apply specifically to `encoding_for_model`?

Reply:
Doing that will result in `import/no-duplicates` error

Reply:
This is what I mean:
```ts
import {
	TiktokenModel,
	// eslint-disable-next-line camelcase
	encoding_for_model,
} from '@dqbd/tiktoken';
```